app/views/homescreen/blocks/_homescreen_block.html.erb

```diff
@@ -1,12 +1,21 @@
 <h3 class="widget-box--header" id="proto-plugin-block">
   <span class="icon-context icon-cart"></span>
-  <span class="widget-box--header-title">Proto Plugin</span>
+  <span class="widget-box--header-title"><%= l(:proto_plugin_name) %></span>
```

Comment:
Wherever you can, please use the standard Rails I18n `I18n.t` helper (available as just `t` in all views) instead of `l`, which comes from the `lib/redmine/i18n.rb` which does some magic with the translation function we want to deprecate further down the road.

There is no difference between `l(:label...)` and `t(:label...)`.